code-challenges.js

```diff
@@ -31,38 +44,77 @@ const temp = 42
 // const temp = 212
 // Expected output: "212 is at boiling point"
 
-// --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
+ //--------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Describe your process:
+// Describe your process: 1.I created two variables for the two different arrays
+//2.I console log with the concat method to combine the arrays
+//3.I console log the first variable since they are combined and used the length method to show how many indexes their are in the array.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+let padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+let padres1998WorldSeriesRuns = [6, 3, 5, 3]
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(padres1998WorldSeriesRuns.length)
```

Comment:
What is this currently returning?  Your approach is right there with concating the variables, but you are only running .length on the second variable.  You can however chain multiple methods together which should give you the desired result.